### PR TITLE
Remove manual LDAP search pagination on UGM principal ``search`` call…

### PR DESCRIPTION
**Pull request by rnixx**
…s. This is done in downstream API as of ``node.ext.ldap`` 1.0b7.

Relates to https://github.com/bluedynamics/node.ext.ldap/pull/36

@jensens @pbauer @fredvd @mauritsvanrees May you be so kind to check the changes?

### CODE REVIEW

File: TODO.rst [hunk 1]
```diff
@@ -2,35 +2,20 @@
...
+[ ] - add/delete users
+[ ] - add/delete groups
+[ ] - add flags for readonly groups and users
+[ ] - modes for only groups or only users from ldap
+[ ] - SSL/TLS configuration TTW
+[ ] - creation defaults TTW
+[ ] - group in group (depends on: node.ext.ldap: group.groups support)
+[ ] - roles from ldap
+[ ] - Option on LDAP inspector whether to use query filters from users and
+      groups config
```
Comment (mauritsvanrees) [inline TODO.rst]: This is MarkDown syntax, which PyPI does not support. In this case, for the last line it gives '(ERROR/3) Unexpected indentation.' Also, for MarkDown it should be the other way around: `- [ ]`.

You can use `restview TODO.rst` to check the ReStructuredText parsing of this file. Or `restview --long-description` to check the long description from `setup.py` that would end up on PyPI.
See https://pypi.org/project/restview/

Comment (mauritsvanrees) [reply TODO.rst]: The `longdescription` command from `zest.releaser` works too, and is easier to type, but I prefer `restview` these days.

File: src/pas/plugins/ldap/cache.rst [hunk 2]
```diff
@@ -8,7 +8,8 @@ Setup
 Basics
 ------
 
-::
+.. code-block:: pycon
```
Comment (mauritsvanrees) [inline src/pas/plugins/ldap/cache.rst]: Shouldn't `pycon` be `python`? Hm, it works though when I try it with `restview`, and when I use a random string instead it gives an error. So `pycon` is apparently okay.
I see very minor differences. Is `pycon` maybe meant as variant that shows better in presentations, for example on the PyCon conference? Do you know that? Just curious. :-)

Comment (rnixx) [reply src/pas/plugins/ldap/cache.rst]: pycon is python console

File: TODO.rst (hunk 1 above)
Comment (rnixx) [reply TODO.rst]: I just finish the work on node.ext.ldap 1.0b7, will fix with the next commit defining the node.ext.ldap dependency version

**Comment by rnixx** [comment]
node.ext.ldap 1.0b7 is released. @mauritsvanrees from my POV we can merge and release.
Should I? Or you? Anyone?

File: src/pas/plugins/ldap/cache.rst (hunk 2 above)
Comment (mauritsvanrees) [reply src/pas/plugins/ldap/cache.rst]: Ah! Thanks for the info.

**Pull request merged**